production/README.md

```diff
@@ -53,7 +53,7 @@ orchestrator from HashiCorp.
 ## Using Helm to deploy on Kubernetes
 
 Here are the Helm charts used to deploy Loki and Promtail to Kubernetes:
-- [Loki](./production/helm/loki/README.md) 
+- [Loki](../production/helm/loki/README.md#loki) 
```

Comment:
```suggestion
- [Loki](./helm/loki/README.md#loki) 
```
Going up a level to go back into the same level is a bit confusing to me. Would this be more succinct?

Reply:
It would, didn't notice the obvious choice at first haha. Thanks again.